docs/sources/panels-visualizations/visualizations/time-series/index.md

```diff
@@ -199,22 +203,46 @@ Set a fixed width of the axis. By default, Grafana dynamically calculates the wi
 
 By setting the width of the axis, data with different axes types can share the same display proportions. This setting makes it easier for you to compare more than one graph’s worth of data because the axes are not shifted or stretched within visual proximity to each other.
 
-### Soft min and soft max
+### Show grid lines
 
-Set a **Soft min** or **soft max** option for better control of Y-axis limits. By default, Grafana sets the range for the Y-axis automatically based on the dataset.
+Set the axis grid line visibility.
 
-**Soft min** and **soft max** settings can prevent blips from turning into mountains when the data is mostly flat, and hard min or max derived from standard min and max field options can prevent intermittent spikes from flattening useful detail by clipping the spikes past a specific point.
+- **Auto:** Automatically shows grid lines based on the density of the data.
+- **On:** Always show grid lines.
+- **Off:** Never show grid lines.
 
-To define hard limits of the Y-axis, You can set standard min/max options. For more information, refer to [Configure standard options][].
+### Color
 
-![Label example](/static/img/docs/time-series-panel/axis-soft-min-max-7-4.png)
+Set the color of the axis
+
+- **Text:** Set the color based on theme text color.
+- **Series:** Set the color based on the series color.
+
+### Show border
+
+Set the axis border visibility.
 
 ### Scale
 
 Set the Y-axis values scale.
 
 - **Linear:** Divides the scale into equal parts.
 - **Logarithmic:** Use a logarithmic scale. When you select this option, a list appears for you to choose a binary (base 2) or common (base 10) logarithmic scale.
+- **Symlog:** Use a symmetrical logarithmic scale. When you select this option, a list appears for you to choose a binary (base 2) or common (base 10) logarithmic scale. The linear threshold option allows you to set the threshold at which the scale changes from linear to logarithmic.
+
+### Centered zero
+
+Set the Y-axis to be centered on zero.
+
+### Soft min and soft max
+
+Set a **Soft min** or **soft max** option for better control of Y-axis limits. By default, Grafana sets the range for the Y-axis automatically based on the dataset.
+
+**Soft min** and **soft max** settings can prevent blips from turning into mountains when the data is mostly flat, and hard min or max derived from standard min and max field options can prevent intermittent spikes from flattening useful detail by clipping the spikes past a specific point.
```

Comment:
"Blip" is a bit colloquial. Would this work? 

I'm not entirely sure how the two parts of this sentence go together. Generally this structure is meant to show a contrast. Also this sentence is super long, so I've tried to chop it up and add some contrast wordingI'm not sure if that works though.

```suggestion
**Soft min** and **soft max** settings can prevent small variations from looking very large when the data is mostly flat. In contrast, hard min or max values derived from standard min and max field options help prevent intermittent spikes from flattening useful detail by clipping the spikes past a specific point.
```